### ISSUE TITLE
CleverTapInboxMessage does not have a property "jsonString"

### ISSUE THREAD

**Issue by malua** [CleverTap/clevertap-cordova#193]
In CleverTapPlugin.m you are accessing the property jsonString of a CleverTapInboxMessage in the method messageDidSelect

```
- (void)messageDidSelect:(CleverTapInboxMessage *_Nonnull)message{
    
    NSString *jsonString = message.jsonString;
    
    if (jsonString != nil) {
        NSString *js = [NSString stringWithFormat:@"cordova.fireDocumentEvent('onCleverTapInboxItemClick', %@);", jsonString];
        [self.commandDelegate evalJs:js];
    }
}
```

But the CleverTapInboxMessage only has property called json. 
This is preventing xcode from building our app. It would be great if that could be fixed! Thanks!

**Comment by darshanclevertap** [comment]
@malua We'll get this checked. Thanks for pointing this out.

**Comment by AishwaryaNanna** [comment]
Hi @malua ,
We have released this update with v2.6.1 ,
Thank you.
